src/Components/SearchForm/SearchForm.jsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from "react";
+import { buttonStyle, containerStyle, inputStyle } from "./styles";
+
+export const SearchForm = ({ initialSearchQuery, onSearch }) => {
+    const [searchValue, setSearchValue] = useState(initialSearchQuery);
+
+    const handleSearch = () => {
+        onSearch(searchValue);
+    }
+
+    const onEnterKey = (e) => {
+        if (e.key === 'Enter') {
+            handleSearch();
+        }
+    }
+
+    return (
+        <div style={containerStyle}>
+            <input 
+                type='text' 
+                onChange={(e) => setSearchValue(e.target.value)} 
+                value={searchValue} 
+                onKeyDown={onEnterKey} 
```

Comment:
Instead of `onKeyDown` here and `onClick` on a `button` below you can wrap these 2 elements in a `form` and set `onSubmit` there.